version.go

```diff
@@ -14,4 +14,4 @@
 
 package lksdk
 
-const Version = "2.0.1"
+const Version = "2.0.5"
```

Comment:
forgot to bump for the last release. So, looks like users are on 2.0.1. Will tag again once this is merged with v2.0.5